packages/fuel-gauge/src/payable-annotation.test.ts

```diff
@@ -41,18 +35,23 @@ test('allow sending coins to payable functions', async () => {
 });
 
 test("don't allow sending coins to non-payable functions", async () => {
+  using contract = await launchPayableContract();
+  const baseAssetId = contract.provider.getBaseAssetId();
+
   // This should fail because the function is not payable
-  await expect(async () =>
-    contract.functions
+  expect(async () => {
```

Comment:
```suggestion
  await expect(async () => {
```

Reply:
Is this still necessary given we are [awaiting the function call](https://github.com/FuelLabs/fuels-ts/pull/2579/files/35d2e65cd5cd24865fa20075b998d002e9b28f30#diff-e65f8549c909083e39ab1251dbf497633e5b4c4301cdbe3c31edcbc4b3473a23R43) ?

Reply:
Yep, because `.rejects.toThrowError` returns a promise:
- https://vitest.dev/api/expect.html#tothrowerror